R/database.R

```diff
@@ -137,6 +155,12 @@ get_ssm_by_samples = function(these_sample_ids,
       maf_df_merge = fread_maf(full_maf_path) %>%
         dplyr::filter(Tumor_Sample_Barcode %in% these_sample_ids) %>%
         dplyr::filter(t_alt_count >= min_read_support)
+      #subset maf to only include first 43 columns (default)
+      if(basic_columns){maf_df_merge = dplyr::select(maf_df_merge, c(1:45))}
+      #subset maf to a specific set of columns (defined in maf_cols)
+      if(!is.null(maf_cols) && !basic_columns){maf_df_merge = dplyr::select(maf_df_merge, all_of(maf_cols))}
+      #print all available columns
```

Comment:
If this was implemented to save time the way it is set up probably doesn't help much because it still loads the full MAF. We need a way to "peek" at (i.e. head) the file for whatever approach we use to get the column names

Reply:
Exactly

Reply:
The problem though is we need the NUMERIC columns to give to cut because it can't select the columns based on their names. We can leave this as a future feature.